features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -3516,6 +3516,7 @@
             </HTTPProxy>
             <!-- Configuration for allowing users to get Previous KID value generated using SHA-1 algo from the JWKS endpoint -->
             <AddPreviousVersionKID>{{oauth.jwks_endpoint.add_previous_version_kid}}</AddPreviousVersionKID>
+            <IsHexifyRequired>{{oauth.jwks_endpoint.is_hexify_required}}</IsHexifyRequired>
```

Comment:
this is only applicable for the x5t representation right? shall we change the name to something more specific. ex: IsThumbprintHexifyRequired

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/5393/commits/4b47cf1ce293e791eebf879fd79d2e5d7d3743b4